cookie-banner-rules-list.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "./CookieBannerRuleList.schema.json",
```

Comment:
One naive question here though, why do we need to add `$schema` here?

Reply:
That's for editors to know which schema to validate the file against. E.g. vscode has some tooling for this. It's not used by the validation script.